server/memphis_handlers_monitoring.go

```diff
@@ -581,14 +581,33 @@ cleanup:
 			logSource, logType = splittedSubj[1], splittedSubj[2]
 		}
 
-		data := string(msg.Data)
-		resMsgs = append(resMsgs, models.Log{
-			MessageSeq: int(msg.Sequence),
-			Type:       logType,
-			Data:       data,
-			Source:     logSource,
-			TimeSent:   msg.Time,
-		})
+		filterSubjectType := strings.Split(cc.FilterSubject, ".")
+		var filterSub string
+		if len(strings.Split(cc.FilterSubject, ".")) == 2 {
```

Comment:
why not use filterSubjectType here?
```suggestion
		if len(filterSubjectType) == 2 {
```